roofit/roofitcore/test/testRooPolyFunc.cxx

```diff
@@ -0,0 +1,107 @@
+// Author: Rahul Balasubramanian, CERN  12/2021
+
+#include "RooPolyFunc.h"
+
+#include "RooPolynomial.h"
+#include "RooRealVar.h"
+#include "RooFormulaVar.h"
+#include "RooWrapperPdf.h"
+
+#include <initializer_list>
+#include <numeric>
+#include <string>
+#include <iostream>
+
+#include "gtest/gtest.h"
+
+namespace {
+
+RooPolyFunc makePolyFunc1D(RooRealVar const &x)
+{
+   // 1 + x + x^2
+   RooPolyFunc polyfunc("f", "f", RooArgSet(x));
+   polyfunc.addTerm(1.0, x, 0);
+   polyfunc.addTerm(1.0, x, 1);
+   polyfunc.addTerm(1.0, x, 2);
+
+   return polyfunc;
+}
+
+RooPolyFunc makePolyFunc2D(RooRealVar const &x, RooRealVar const &y)
+{
+   // 1 + x + y + x^2 + y^2
+   RooPolyFunc polyfunc("f", "f", RooArgSet(x, y));
+   polyfunc.addTerm(1.0, x, 0, y, 0);
+   polyfunc.addTerm(1.0, x, 0, y, 1);
+   polyfunc.addTerm(1.0, x, 1, y, 0);
+   polyfunc.addTerm(1.0, x, 2, y, 0);
+   polyfunc.addTerm(1.0, x, 0, y, 2);
+
+   return polyfunc;
+}
+} // namespace
+
+using Doubles = std::initializer_list<double>;
+
+TEST(RooPolyFunc, WrappedPdfClosure)
+{
+   RooRealVar x("x", "x", 0., -10., 10.);
+   RooPolyFunc polyfunc = makePolyFunc1D(x);
```

Comment:
I see RooPolyFunc misses an assignment operator. This could be the cause of the error seen in the Windows test. Can you try implementing the assignment operator ?

Reply:
Thank you Lorenzo, that must be it! I will implement the assignment operator in a follow-up PR to re-enable the Windows tests.